.github/workflows/mucoll-rebuild.yml

```diff
@@ -0,0 +1,77 @@
+name: Re-build and publish MuColl
```

Comment:
I know this works in your fork, but I don't understand how. Don't you first need to build the base image everytime as the `FROM` base image tag depends on the SHA of this commit?

Reply:
It's the branch name, not the SHA. But don't we need something to trigger the building of the first `mucoll-spack` image in any new branch?

Reply:
Can I propose that we disable the partial image build now and only enable the full image build for now? It will make testing changes that only affect the last image easier, since the partial builds fail by design.

We then revisit this once we have a series of central base images that branches can start from.

Reply:
I'm pretty open to any problem, but can you elaborate a bit more? 
I don't think I understand: 
-  what the problem is with the current workflow
-  why only doing the full build would make it easier to debug things